core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -3809,6 +3809,53 @@ llvm_jit_free_frame(WASMExecEnv *exec_env)
 #endif /* end of WASM_ENABLE_DUMP_CALL_STACK != 0 \
           || WASM_ENABLE_PERF_PROFILING != 0 */
 
+#if WASM_ENABLE_GC != 0
+void *
+llvm_jit_create_func_obj(WASMModuleInstance *module_inst, uint32 func_idx)
+{
+    WASMModule *module = module_inst->module;
+    WASMRttTypeRef rtt_type;
+    WASMFuncObjectRef func_obj;
+    WASMFunctionInstance *func_inst;
+    WASMFuncType *func_type;
+    uint32 type_idx;
+
+#if WASM_ENABLE_JIT != 0
+    if (Wasm_Module_AoT == module_inst->module_type) {
+        return aot_create_func_obj(module_inst, func_idx);
+    }
+#endif
```

Comment:
Just `bh_assert(module_inst->module_type == Wasm_Module_Bytecode)`, the similar codes in this file have been modified in main branch.